DigitalLearningSolutions.Web/Styles/trackingSystem/delegates/emailDelegates.scss

```diff
@@ -5,3 +5,7 @@
   border-radius: 4px;
   margin-right: nhsuk-spacing(2);
 }
+
+.delegate-label {

```

Comment:
This fix won't work as expected in IE, use the word-break class on the elements (This is defined in index.scss)